app/MainHie.hs

```diff
@@ -133,10 +133,12 @@ run opts = do
 
   let initOpts = defaultCradleOpts { cradleOptsVerbosity = verbosity }
       verbosity = if optBiosVerbose opts then Verbose else Silent
+      biosLogLevel = if optBiosVerbose opts then L.DEBUG else L.INFO
       -- Running HIE on projects with -Werror breaks most of the features since all warnings
       -- will be treated with the same severity of type errors. In order to offer a more useful
       -- experience, we make sure warnings are always reported as warnings by setting -Wwarn
 --      ghcModOptions = defaultOpts { GM.optGhcUserOptions = ["-Wwarn"] }
+  Core.setupLogger mLogFileName ["hie-bios"] biosLogLevel
```

Comment:
@bubba Do you know what this line actually does? By bisecting I concluded that this line is responsible for the failing test-cases. 
I intended to enable hie-bios debug output, but maybe I am misusing this `Core.setupLogger`?

Reply:
The Haddock documentation on this isn't exactly enlightening:

https://github.com/alanz/haskell-lsp/blob/5d1dfe82c9955d81cc0e0bb7a3e8bf6d2c4c42e4/src/Language/Haskell/LSP/Core.hs#L945-L966

But this is a wrapper for [hslogger](https://hackage.haskell.org/package/hslogger) as far as I was aware. It could be possible that something is going wrong when opening two logs to the same file at once, but I have no idea as to why this would cause the liquid tests to fail?

Reply:
Literally no clue why this is failing. Afaik it does exactly what I want it to do. Local tests also dont fail.

Reply:
The problem is fixed if we change `mLogFileName` to `fmap (++ ".bios") mLogFileName`. So, indeed, the problem is caused by Core.setupLogger, potentially by opening the File Handle twice. 

How can we work around this? What we want is to set the logging verbosity of two different logging components independently. However, it would be great if the logging output could be to the same file. Does haskell-lsp provide any functionality for it?
cc @mpickering